extensions/investspec/test/test_module/test_module/test_module.py

```diff
@@ -0,0 +1,196 @@
+import pint
```

Comment:
This would be interesting to get feedback from more folks too (@phargogh, @dcdenu4, @jagoldstein?) the way I did it originally of course looks more natural to me, but I'm open to changing it. Currently, most UG pages have the args in a bulleted list, where each name is bolded and the description follows on the same line.

Reply:
I'm glad this is being addressed and I don't have a strong opinion either way, other than I want there to be a convention that is consistent, because right now there is not. That said, I do think think it's helpful to have some way of clearly distinguishing between the term and it's definition, like bolding the name or a line break, but probably not both. If I had to vote, I'd go for the boldface approach.

(Another point re: Dave's example, currently our unofficial [Style Guide](https://docs.google.com/document/d/1BHwHDu_I-x0s_2GsbUb4rfVmXMkl7kl97sx2suBTLh8/edit?usp=sharing)'s last bullet suggests adopting Google's developer standard of abbreviating common units of measure (e.g.: "mm" instead of "millimeters"), but more importantly, let's please just choose one rule for writing out units of measure, document it in the Style Guide, and stick with it in the UG (maybe this point is already being discussed elsewhere).)

Reply:
An example of how it is now (ignore the different description text):
![image](https://user-images.githubusercontent.com/43770515/126241133-c3814610-0075-464b-9615-6278c20433ab.png)

Two different options for the new format:
![image](https://user-images.githubusercontent.com/43770515/126241195-0fcc45ff-8b9f-4c32-9a49-3fcb0704ead8.png)
![image](https://user-images.githubusercontent.com/43770515/126241218-61df15b8-01fa-40c0-8c0b-af8865d931f0.png)

Reply:
Thanks for reminding us of the Style Guide @jagoldstein! In the input data description anyway, I'm inclined to spell out the units because (1) it's a little easier because it's the default that `pint` gives us and (2) occasionally I find unit abbreviations that I'm unsure of. They're probably hopefully standard though...

Reply:
I prefer new format option 2, but again, I'm just happy we're standardizing this in some way.
Thanks for all your work on this!

Reply:
Ok, I'm down for spelling out unit names, especially in the description! It seems more accessible that way. I'll update the Style Guide to reflect that, and then we'll I believe we'll have lots of unit names to update in the UG!

But what about if they are repeated later? Do we define the abbreviation and then use that, or do we spell them out every time?

Reply:
I don't think I have much of a preference with the formats, but do have a suggestion that is probably too verbose:

**Analysis Year** (optional) *input-type*: number, *units*: year
**Precipitation** (required) *input-type*: raster, *units*: millimeters / year

I like labeling what things are so they take any guesswork out. But, that's just my preference.

Reply:
I sort of agree with Doug's point here about labelling. My suggestion tried to solve the same problem - taking the guesswork out of what those three items inside the parens are - not by adding labels but by placing them in more distinct areas: inside vs outside the parens.

Don't care too much about whether there's a newline for the description.

Reply:
Oooo, yeah I like Doug's point too... be explicit and remove as much guesswork as possible.

Reply:
For the sake of a technical manual, explicit is probably better as a general rule.

Reply:
I like the labels!